src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1325,6 +1325,7 @@ function ($a) {
                                 ->fixXmlConfig('option')
                                 ->children()
                                     ->scalarNode('dsn')->end()
+                                    ->booleanNode('reset_on_message')->defaultFalse()->info('Reset the container services after each message.')->end()
```

Comment:
Having a setting for that feature makes the whole thing very fragile. I'm wondering if we should not make a specific fix for Monolog instead.

Reply:
This is not only for monolog, it's also for doctrine, cache, etc.

This setting exists in swarott for 1.5 years, and before that I had a custom listener that does that for ages. Never had a problem :)

Reply:
What I meant is: do we really need a setting or wouldn't it be better to always reset the container?

Reply:
Ah got it. I always enable my listener, so 👍🏼 for enabling it by default.

Reply:
Oh, I just realized something: We could not reset the container after each message, because when the transport is sync you dont't want it.

It's only for async transport

So I think `false` is a nice default.

Reply:
Can you change the info message to give a hint about when to turn it on/off?